File: fixtures/conformance/java/deep_nesting.java
public class deep_nesting {
    public static int deep_nesting(int[] items) {
        for (int a : items) {
            if (a > 0) {
                while (a > 0) {
                    try {
                        if (a > 0) {
                            return a;
                        }
                    } finally {
                    }
                }
            }
        }
        return 0;
    }
}

File: fixtures/conformance/kotlin/deep_nesting.kt
// Anidamiento profundo (5 niveles: for>if>while>try>if). try/finally anida SIN ser
// decision (analogo del `with` de Python; catch_block seria la decision).
fun deep_nesting(items: IntArray): Int {
    for (a in items) {
        if (a > 0) {
            while (a > 0) {
                try {
                    if (a > 0) {
                        return a
                    }
                } finally {
                }
            }
        }
    }
    return 0
}
